app/views/govuk_publishing_components/components/_date_input.html.erb

```diff
@@ -2,11 +2,11 @@
   id ||= "input-#{SecureRandom.hex(4)}"
   name ||= nil
   items ||= [
-    { :name => "day", :width => 2 },
-    { :name => "month", :width => 2 },
-    { :name => "year", :width => 4 }
+    { :name => "day", :width => 2, autocomplete: "bday-day" },
```

Comment:
I think I'd rather see these autocomplete strings only added in the case where the date is a DOB and autocomplete is on, rather than added here and removed in many cases. It feels more expandable if we were to ever have alternate autocomplete patterns, and generally cleaner.

Reply:
I think I've addressed the changes and have moved the autocomplete values to where the component options are passed in -

`autocomplete: autocomplete_date_of_birth ? "bday-#{item[:name]}" : nil`

Added changes under a second temporary commit.